schoolopy/main.py

```diff
@@ -1977,15 +1984,15 @@ def send_message(self, subject, message, user_ids):
         :param user_ids: A list of user ids to send the message to.
         """
         recipients = ','.join([str(uid) for uid in user_ids])
-        return Message(self._post('messages', {'subject': subject, 'message': message.json(), 'recipient_ids': recipients}))
```

Comment:
Don't remove this `.json()`

Reply:
In this case, the message is just text to put into it. I should probably make the parameter "message_text" now that I think about it.